src/components/ButtonGroupPicker/styled/label.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { FONT_SIZE_TEXT_MEDIUM } from '../../../styles/fontSizes';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+const StyledLegend = attachThemeAttrs(styled.legend)`
+    border: 0;
+    padding: 0;
+    display: inline-block;
+    color: ${props => props.palette.text.label};
+    font-size: ${FONT_SIZE_TEXT_MEDIUM};
+    line-height: 1.5;
+    margin: 0 auto 0.1rem auto;
+    text-align: center;
+    box-sizing: border-box;
+    ${props =>
+        props.variant === 'inverse' &&
+        `
+            color: ${props.palette.isDark ? '#576574' : 'rgb(178,178,178)'}
```

Comment:
we can't use colors inline, all colors must come from theme, here you can use some of the text variants in the theme like ```props.palette.text.title``` or other that match with the design color